src/counter/src/azurefiles/counterService/Service Resources/service.yaml

```diff
@@ -1,26 +1,28 @@
-﻿## Service definition ##
-application:
-  schemaVersion: 1.0.0-preview1
-  name: counterApp
-  properties:
-    services:
-      - name: counterService
-        properties:
-          description: A web service that serves the counter value stored in the Azure Files volume.
-          osType: Windows
-          codePackages:
-            - name: counterService
-              image: counterservice:dev
-              endpoints:
-                - name: counterServiceListener
-                  port: 8080
-              environmentVariables:
-                - name: ASPNETCORE_URLS
-                  value: http://+:8080
-              resources:
-                requests:
-                  cpu: 0.5
-                  memoryInGB: 0.5
-          replicaCount: 1
-          networkRefs:
-            - name: counterAppNetwork
\ No newline at end of file
+﻿## Service definition ##
+application:
+  schemaVersion: 1.0.0-preview1
+  name: counterApp
+  properties:
+    services:
+      - name: counterService
+        properties:
+          description: A web service that serves the counter value stored in the Azure Files volume.
+          osType: Windows
+          codePackages:
+            - name: counterService
+              image: counterservice:dev
+              endpoints:
+                - name: counterServiceListener
+                  port: 8080
+              environmentVariables:
+                - name: ASPNETCORE_URLS
+                  value: http://+:8080
+              resources:
+                requests:
+                  cpu: 0.5
+                  memoryInGB: 0.5
+          replicaCount: 1
+          networkRefs:
+            - name: counterAppNetwork
+              endpointRefs:
```

Comment:
endpointsRef format is for the preview2 schema, not require in preview1

Reply:
Thanks for catching this.

Reply:
Thanks for catching this.